scripts/release.sh

```diff
@@ -24,27 +24,53 @@ release_app () {
     APP_NAME=$1
     COMPOSE_PATH=$2
     AETHER_APP="aether-${1}"
-    echo "version: $VERSION"
 
     echo "Building Docker image ${IMAGE_REPO}/${AETHER_APP}:${VERSION}"
     docker-compose -f $COMPOSE_PATH build \
         --build-arg GIT_REVISION=$TRAVIS_COMMIT \
         --build-arg VERSION=$VERSION \
         $APP_NAME
 
-    docker tag ${AETHER_APP} "${IMAGE_REPO}/${AETHER_APP}:${VERSION}"
-    docker tag ${AETHER_APP} "${IMAGE_REPO}/${AETHER_APP}:latest"
-
     echo "Pushing Docker image ${IMAGE_REPO}/${AETHER_APP}:${VERSION}"
+    docker tag ${AETHER_APP} "${IMAGE_REPO}/${AETHER_APP}:${VERSION}"
     docker push "${IMAGE_REPO}/${AETHER_APP}:${VERSION}"
-    docker push "${IMAGE_REPO}/${AETHER_APP}:latest"
+
+    if [[ $VERSION != "alpha" ]]
+    then
+        echo "Pushing Docker image ${IMAGE_REPO}/${AETHER_APP}:latest"
+        docker tag ${AETHER_APP} "${IMAGE_REPO}/${AETHER_APP}:latest"
+        docker push "${IMAGE_REPO}/${AETHER_APP}:latest"
+    fi
 }
 
+# release version depending on TRAVIS_BRANCH/ TRAVIS_TAG
+if [[ $TRAVIS_TAG =~ ^[0-9]+\.[0-9]+[\.0-9]*$ ]]
+then
+    VERSION=$TRAVIS_TAG
+
+elif [[ $TRAVIS_BRANCH =~ ^release\-[0-9]+\.[0-9]+[\.0-9]*$ ]]
+then
+    # remove "release-" prefix and add "-rc" suffix
+    VERSION=${TRAVIS_BRANCH#"release-"}-rc
```

Comment:
This line will remove the `release-` prefix on the branch name... would it make the trick for you, actually the only difference with the current version is that we loose the last number of the version. Now `0.9.x-rc` with this change `0.9-rc`

Reply:
Right, that's the problem, and that's why we have the VERSION file after all - it would be better to have `0.9.x-rc`

Reply:
Reintroduced file! :wink:

Reply:
Ideally, the script would write an incremented version to the VERSION file once it's done a release, so that subsequent RCs are correct.

Reply:
If you don't mind I would prepare the AUTOMATIC incremented version in another PR...